python/tidy/servo_tidy_tests/test_tidy.py

```diff
@@ -221,6 +223,40 @@ def test_file_list(self):
         lst = list(file_list)
         self.assertEqual([os.path.join(base_path, 'whee', 'test.rs')], lst)
 
+    def test_commit_messages(self):
```

Comment:
Yes, it's quite a bit of hack. This is exactly why I chose to avoid having a test for this - a similar reason why `FileList(only_changed_files=True)` doesn't have a test. Should we really have this test just to make sure that we're doing it right? I'm fine with blindly trusting those two simple subprocess calls :neutral_face: